spec/models/job_application_spec.rb

```diff
@@ -314,6 +314,110 @@
     end
   end
 
+  describe "#can_jobseeker_initiate_message?" do
+    subject { job_application.can_jobseeker_initiate_message? }
+
+    context "when status allows jobseeker to initiate messages" do
+      %w[interviewing unsuccessful_interview offered declined].each do |allowed_status|
+        context "when status is #{allowed_status}" do
+          let(:job_application) { create(:job_application, status: allowed_status) }
```

Comment:
Suggestion: Use `build_stubbed` instead of `create` for these tests if possible, since we don't need the record in DB to test the method's logic and it is faster?